HAL/picnc.h

```diff
@@ -50,11 +50,11 @@ typedef enum {
 #define SPIBUFSIZE		20		/* SPI buffer size */
 #define BUFSIZE			(SPIBUFSIZE/4)
 
-#define STEPBIT			22		/* bit location in DDS accum */
+#define STEPBIT			16		/* bit location in DDS accum */
```

Comment:
The max speed step frequency is not limited by those settings. In any case I put your algorithm back. Stand by for another pull request. 

Sent from my iPhone

> On Dec 21, 2015, at 3:51 PM, GP Orcullo <notifications@github.com> wrote:
> 
> In HAL/picnc.h:
> 
> > @@ -50,11 +50,11 @@ typedef enum {
> >  #define SPIBUFSIZE		20		/* SPI buffer size */
> >  #define BUFSIZE			(SPIBUFSIZE/4)
> >  
> > -#define STEPBIT			22		/* bit location in DDS accum */
> > +#define STEPBIT			16		/* bit location in DDS accum */
> You are sacrificing step resolution over speed
> 
> —
> Reply to this email directly or view it on GitHub.
>